golang/sqlboiler/graph/templates/singleton/schema.tpl

```diff
@@ -18,10 +18,10 @@ type Query {
 {{- $modelNamePlural := $table.Name | plural | titleCase -}}
 {{- $modelNameCamel := $tableNameSingular | camelCase}}
 
-  all{{$modelNamePlural}}(
-    since: ID
+  search{{$modelNamePlural}}(
+    pageNumber: Int
```

Comment:
when dataset grows, doing `offset` pagination can get visibly sluggish. on the other hand, pagination via `since` (e.g. implementing with `"id > ?", since`) remains equally fast; even for very large tables

whenever `1, 2, 3... 42` style pagination is not explicitly required, i'd try to get away with providing `next page` style pagination for [performance reasons and (arguably speed is also) usability reasons](https://www.citusdata.com/blog/2016/03/30/five-ways-to-paginate/)

can i ask to continue providing `since: ID` (or `sinceID: ID`) alongside `pageNumber`? if both are given, then `pageNumber` is ignored 🙇

Reply:
TIL! 🙏🏻 Will do!

Reply:
def4f56ee7152366faf57bf93755d4ee47de466f 🙇